tests/rkt_ace_validator_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build ignore
+// +build host coreos src
```

Comment:
I guess we can simply remove this build tag, because we want this test to be executed on all flavors.